shared/config.json

```diff
@@ -2,6 +2,7 @@
     "enable_geocoding": true,
     "include_census_data": true,
     "_backend": "Configuration used by the backend.",
+    "include_census_data_for_representative_points": false,
```

Comment:
Maybe `census_data_level` which is either `representative_point` or `service_area`?